### ISSUE TITLE
Support for excluding mutators

### ISSUE THREAD

**Issue by npathai** [hcoles/pitest#954]
Carrying over this feature request from https://github.com/szpak/gradle-pitest-plugin

Issue:
[Support for excluding mutators](https://github.com/szpak/gradle-pitest-plugin/issues/187)

Feature Request:
Ability to specify the mutators like this:

```
pitest {
    // …
    mutators = ['All'] - ['UOI']
    // …
}
```
or, if that is not possible, with something equivalent, like this:

```
pitest {
    // …
    mutators = ['All']
    excludeMutators = ['UOI']
    // …
}
```
Use case: instead of having a long list of all mutators to run, be able to specify a large set of mutators, like 'ALL', and then just exclude some from the list, like ['UOI'], when they produce too many false positives.

I had gone through the source code of PIT and found that there is no way to exclude specific mutators.

**Comment by hcoles** [comment]
Hi @npathai,

I think something like this would be useful. The easiest syntax to implement would be

mutators = ['ALL', '-UOI', '-CONDITIONAL_BOUNDARY'] 

This would work without updating any of the tooling.

That said, unless you are doing research, enabling ALL mutators and then disabling one or two of them is not a good idea. Most of the mutators in the 'RV' research set are not usable for day to day development. I will likely move them out of the main codebase now that pitest supports pluggable mutators.

**Comment by hcoles** [comment]
Released in 1.7.3
